src/main/java/dk/cngroup/lentils/repository/CypherRepository.java

```diff
@@ -11,5 +11,6 @@
     Cypher findByStage(int stage);
     Cypher findFirstByStageGreaterThanOrderByStageAsc(int stage);
     List<Cypher> findAllByOrderByStageAsc();
+    List<Cypher> findByStageGreaterThan(int stage);
```

Comment:
Tuhle metodu bych čistě pro jistotu pojmenoval jako: findByStageGreaterThanOrderByStageAsc
Ať to ten seznam seřadí podle stage.

Reply:
hotovo